### ISSUE TITLE
🐛 BUG: Extension not working on Vivaldi 7.0

### ISSUE THREAD

**Issue by Karbust** [dailydotdev/daily#1572]
### What went wrong? 🤔

The extension page is just black.

![image](https://github.com/user-attachments/assets/14927fe5-a9b7-4900-9b91-fb3b157f7309)

### Expected Behavior

The extension page to be visible.

### Steps to Reproduce Issue

```bash
1. Install Vivaldi 7.0
2. Install the Daily.dev extension
3. Open it
```

### Environment

Vivaldi 7.0.3495.10 (Stable channel) (64-bit)

### Browsers

Other

### OS

Windows

### Version of daily.dev

3.34.6

### Additional Context

```bash
Opening directly https://app.daily.dev works without issues.
```

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by Abhi-Bohora** [comment]
Hello @Karbust I think daily.dev extension works only on Chrome, opera, and Firefox, I may be wrong. And this `https://app.daily.dev` works regardless of what browser you use because it's not a extension it's a webapp

**Comment by Karbust** [comment]
> Hello @Karbust I think daily.dev extension works only on Chrome, opera, and Firefox, I may be wrong. And this `https://app.daily.dev` works regardless of what browser you use because it's not a extension it's a webapp

I have been using Vivaldi for years, and started using daily.dev almost 2 years ago, always worked until Vivaldi 7.0 that was released this month.

Plus Vivaldi is based on Chromium, it uses the Chrome Store for extensions.

**Comment by rebelchris** [comment]
@Karbust Officially we indeed don't support third party browser as it's nearly impossible to keep up with them.
I've tested the latest Vivaldi release and it seems to work stable there.

![Screenshot 2024-11-06 at 12 31 31](https://github.com/user-attachments/assets/aa521a1b-3920-4518-91f1-c5c8739c2807)

Do you have anything in the console that can shed some light on the issue perhaps?

**Comment by Karbust** [comment]
> @Karbust Officially we indeed don't support third party browser as it's nearly impossible to keep up with them. I've tested the latest Vivaldi release and it seems to work stable there.
> 
> ![Screenshot 2024-11-06 at 12 31 31](https://private-user-images.githubusercontent.com/554874/383516645-aa521a1b-3920-4518-91f1-c5c8739c2807.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA4OTA2MTEsIm5iZiI6MTczMDg5MDMxMSwicGF0aCI6Ii81NTQ4NzQvMzgzNTE2NjQ1LWFhNTIxYTFiLTM5MjAtNDUxOC05MWYxLWM1Yzg3MzljMjgwNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwNlQxMDUxNTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZjVmMzU5ZWU0NzFkMWNjOGM2NDkzMDUwMWZkMDA1ZTZjNTUxYjJiZGJiMzg3MDAwMmE0ZGM2ODYxZDAzMzQ4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.6zac3EYi69hA7ivuxeUI8InUQHscq8IRBYzJ2jXL1Lw)
> 
> Do you have anything in the console that can shed some light on the issue perhaps?

There is absolutely nothing, only the ASCII logo of daily.dev, other than that, black page.

Really strange that this only started happening after Vivaldi's 7.0 release, it was working fine.

This is happening on 2 computers, both have different profiles configured and some different extensions, but same version on both.

**Comment by idoshamun** [comment]
Can you please check the network tab?

Maybe Vivaldi changed their policy in regards to extensions or specifically new tab extensions

**Comment by Karbust** [comment]
> Can you please check the network tab?
> 
> Maybe Vivaldi changed their policy in regards to extensions or specifically new tab extensions

This is what I get on the network tab:

![image](https://github.com/user-attachments/assets/13677a3b-5ee8-464d-a74d-e1d02a3a28be)

I can go to https://api.daily.dev/boot?v=3.34.6 and it loads fast, no idea why it is pending on the extension.

**Comment by idoshamun** [comment]
Probably something vivaldi changed in the new version. it requires to deep dive into their release notes and understand what's going on

**Comment by idoshamun** [comment]
Would love to get the community help for this one 🙏

**Comment by arbazkpathan** [comment]
Try reinstalling Your extinsion
if not working then
remove your browser cache

**Comment by Karbust** [comment]
The issue seems to be resolved in the latest version of Vivaldi

Vivaldi 7.0.3495.18 (Stable channel) (64-bit)

---

> Try reinstalling Your extinsion if not working then remove your browser cache

I had already done that, didn't solve it.

**Comment by idoshamun** [comment]
@Karbust that's fantastic news! can i close the issue now?

**Comment by Karbust** [comment]
> @Karbust that's fantastic news! can i close the issue now?

Yes! Thank you for the assistance.
